tools/tasks/copyChangelog.js

```diff
@@ -0,0 +1,88 @@
+const fs = require('fs');
+const showdown = require('showdown');
+const colors = require('ansi-colors');
+const fancyLog = require('fancy-log');
+const OPTIONS = require('../options');
+
+/**
+ * Loads the changelog markdown into a string, and then returns it.
+ *
+ * @method loadChangelogMarkdown
+ * @returns {String} the raw markdown of the changelog
+ */
+function loadChangelogMarkdown() {
+    const sourceFile = OPTIONS.FILE.CHANGELOG_SOURCE;
+    const sourceMarkdown = fs.readFileSync(sourceFile, { encoding: 'utf8' });
+
+    return sourceMarkdown;
+}
+
+/**
+ * Encapsulates the `makeHtml(String)` function provided by showdown.
+ *
+ * @method parseMarkdown
+ * @param {String} markdown 
+ * @returns {String} parsed html
+ */
+function parseMarkdown(markdown) {
+    const converter = new showdown.Converter({ simpleLineBreaks: true });
+
+    return converter.makeHtml(markdown);
+}
+
+/**
+ * Gets the first `h1` HTML element in the stringified HTML.
+ *
+ * @method getHtmlElement
+ * @param {String} totalHtml
+ * @returns {String} the HTML section
+ */
+function getHtmlElement(totalHtml) {
```

Comment:
why is this function important?  what is this actually doing and why do we need it?